data/models6.sample.go

```diff
@@ -0,0 +1,13 @@
+package models
+
```

Comment:
Add import fmt.

Reply:
That is an old commit and I have not changed it.